staging/src/k8s.io/client-go/rest/request.go

```diff
@@ -391,6 +393,17 @@ func (r *Request) Timeout(d time.Duration) *Request {
 	return r
 }
 
+// MaxRetries makes the request use the given integer as a ceiling of retrying
+// upon receiving "Retry-After" headers and 429 status-code in the response.
+// A zero maxRetries prevent it from doing retires and return an error immediately.
```

Comment:
Mention the default? ("The default is 10 unless this function is specifically called with a different value." or something)

Reply:
mentioned the default value in the doc